src/Processors/QtiV2/In/Interactions/ChoiceInteractionMapper.php

```diff
@@ -90,4 +99,31 @@ private function buildOptions(SimpleChoiceCollection $simpleChoices)
         }
         return $options;
     }
+
+    private function buildFeedbackMetadata(SimpleChoiceCollection $simpleChoices)
+    {
+        /* @var $choice SimpleChoice */
+        $metadata = [];
+        foreach ($simpleChoices as $key => $choice) {
+            $flow = $choice->getContent();
+            $class = new \ReflectionClass(get_class($flow));
+            if(property_exists($flow,'dataPlaceHolder')){
+                $property = $class->getProperty('dataPlaceHolder');
+                $property->setAccessible(true);
+                $feed = $property->getValue($flow);
+                $feedArray = array_values((array) $feed);
+                if(sizeof($feedArray)==2){
+                    $feeddataArray = array_values((array) $feedArray[0]);
+                    $feedarr = array_values((array) $feeddataArray[3][0]);
+                    if($feedarr[2]=='text/html'){
+                        $htmlfile = ConvertToLearnosityService::$inputDir.'/'.$feedarr[1];
```

Comment:
See my comment on the approach here: [#diff-6f0e68dc90c0979d95f5acaf95f64b4eR52](https://github.com/Learnosity/learnosity-qti/pull/29/files#diff-6f0e68dc90c0979d95f5acaf95f64b4eR52)

Reply:
Done . Please check PR #41